xlive/stdafx.cpp

```diff
@@ -7,4 +7,9 @@ std::random_device rd;
 extern "C" {
 	__declspec(dllexport) DWORD NvOptimusEnablement = 0x00000001;
 	__declspec(dllexport) DWORD AmdPowerXpressRequestHighPerformance = 0x00000001;
-}
\ No newline at end of file
+}
+
+extern "C" int LLVMFuzzerTestOneInput(uint8_t* buf, size_t len)
```

Comment:
lol

Reply:
Do you want this removed?
It doesn't do anything atm since the flag isn't passed but it lets you compile with the flag if anyone wants to test things using it.

Reply:
just removed it for now